src/client/layouts/ConsentsLayout.tsx

```diff
@@ -117,7 +119,7 @@ export const ConsentsLayout: FunctionComponent<ConsentsLayoutProps> = ({
             </div>
             <ConsentsBlueBackground>
               <div css={[gridItem(gridItemColumnConsents), controls]}>
-                {!error && (
+                {!error && !hideContinueButton && (
```

Comment:
Used to hide the `Save and continue` button on the `ConsentsLayout`, as we don't want this button to show on the "Welcome"/"Set password" page